.github/workflows/tests.yml

```diff
@@ -40,6 +44,7 @@ jobs:
       env:
         CLEAN_SCHEDULE: "* * * * *"
         REPORTS_SCHEDULE: "* * * * *"
+        REGION_CONFIG: "london"
```

Comment:
Am I right in understanding that the CI/CD tests will use London as the region of choice?  Do we see any issue with this?  FOr instance, are there configurations from other LAs that could disrupt the tests in a way that London does not?  I'm kind of wondering if we design a synthetic region to do testing on instead of using an actual one.  Saying that, London is the biggest and probably the best one to use for testing.

Reply:
Yes the CI/CD tests will use London as the region. Yeah if you didn't make the config correctly then I'm sure some tests would fail. I'm just not sure this is necessarily the place to test if the configs themselves are correct, that should probably be in PRs for that repo. So I think at this point we can maybe safely assume they work ok (in an effort to save resource by not duplicating checks)

Reply:
Have added a test config now